src/triage/predictlist/__init__.py

```diff
@@ -328,8 +245,23 @@ def __init__(self, db_engine, project_path, model_group_id):
             model_storage_engine=ModelStorageEngine(self.project_storage),
             db_engine=self.db_engine,
             replace=True,
-            run_id=self.run_id,
+            run_id=self.experiment_run_id,
         )
+    
+    def get_temporal_config_for_retrain(self, prediction_date):
```

Comment:
since we're only predicting for a specific date, should probably explicitly set the `test_duration` to `0day`?